modules/37_industry/subsectors/sets.gms

```diff
@@ -198,6 +198,13 @@ $endif.process_based_steel
                       feh2_otherInd, fehe_otherInd, feelhth_otherInd)
   /
 
+  in_chemicals_37(all_in)   "chemicals FE that can provide feedstocks"
```

Comment:
A more descriptive name would be welcome.  Somebody sometime will mistake this for all the production factors in the chemicals subsector.